dev-docs/bidders/nexverse.md

```diff
@@ -0,0 +1,61 @@
+
+---
+layout: bidder
+title: Nexverse
+description: Prebid Nexverse Bidder Adapter
+biddercode: nexverse
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+tcf2_supported: true
```

Comment:
This is old. It's now "tcfeu_supported". And you can't claim support for TCFEU without suppplying a GVLID.